README.md

```diff
@@ -168,60 +168,60 @@ will be added in the future.
-| **v0.8.1-alpha**     | v0.15.2-beta | v0.20.1-beta | v0.2.8-alpha  | v0.5.8-alpha | n/a            |
-| **v0.8.0-alpha**     | v0.15.1-beta | v0.20.1-beta | v0.2.8-alpha  | v0.5.8-alpha | n/a            |
-| **v0.7.1-alpha**     | v0.15.0-beta | v0.19.1-beta | v0.2.8-alpha  | v0.5.7-alpha | n/a            |
-| **v0.7.0-alpha**     | v0.15.0-beta | v0.19.1-beta | v0.2.8-alpha  | v0.5.7-alpha | n/a            |
-| **v0.6.7-alpha**     | v0.14.3-beta | v0.18.0-beta | v0.2.7-alpha  | v0.5.6-alpha | n/a            |
-| **v0.6.6-alpha**     | v0.14.3-beta | v0.18.0-beta | v0.2.7-alpha  | v0.5.6-alpha | n/a            |
-| **v0.6.5-alpha**     | v0.14.2-beta | v0.15.1-beta | v0.2.7-alpha  | v0.5.5-alpha | n/a            |
-| **v0.6.4-alpha**     | v0.14.2-beta | v0.15.1-beta | v0.2.7-alpha  | v0.5.4-alpha | n/a            |
-| **v0.6.3-alpha**     | v0.14.2-beta | v0.15.1-beta | v0.2.7-alpha  | v0.5.4-alpha | n/a            |
-| **v0.6.2-alpha**     | v0.14.1-beta | v0.15.1-beta | v0.2.7-alpha  | v0.5.4-alpha | n/a            |
-| **v0.6.1-alpha**     | v0.14.1-beta | v0.15.1-beta | v0.2.7-alpha  | v0.5.2-alpha | n/a            |
-| **v0.5.3-alpha**     | v0.13.3-beta | v0.14.1-beta | v0.2.6-alpha  | v0.5.0-alpha | n/a            |
-| **v0.5.2-alpha**     | v0.13.3-beta | v0.14.1-beta | v0.2.6-alpha  | v0.5.0-alpha | n/a            |
-| **v0.5.1-alpha**     | v0.13.0-beta | v0.14.1-beta | v0.2.6-alpha  | v0.5.0-alpha | n/a            |
-| **v0.5.0-alpha**     | v0.13.0-beta | v0.14.1-beta | v0.2.6-alpha  | v0.5.0-alpha | n/a            |
-| **v0.4.1-alpha**     | v0.12.1-beta | v0.11.4-beta | v0.2.3-alpha  | v0.4.4-alpha | n/a            |
-| **v0.4.0-alpha**     | v0.12.0-beta | v0.11.2-beta | v0.2.3-alpha  | v0.4.3-alpha | n/a            |
-| **v0.3.4-alpha**     | v0.11.1-beta | v0.11.2-beta | v0.2.2-alpha  | v0.3.4-alpha | n/a            |
-| **v0.3.3-alpha**     | v0.11.1-beta | v0.11.2-beta | v0.2.2-alpha  | v0.3.4-alpha | n/a            |
-| **v0.3.2-alpha**     | v0.11.1-beta | v0.11.1-beta | v0.2.2-alpha  | v0.3.4-alpha | n/a            |
-| **v0.3.1-alpha**     | v0.11.1-beta | v0.11.1-beta | v0.2.2-alpha  | v0.3.3-alpha | n/a            |
-| **v0.3.0-alpha**     | v0.11.1-beta | v0.11.0-beta | v0.2.2-alpha  | v0.3.2-alpha | n/a            |
-| **v0.2.0-alpha**     | v0.11.1-beta | v0.10.0-beta | v0.2.1-alpha  | n/a          | n/a            |
-| **v0.1.1-alpha**     | v0.11.0-beta | v0.8.1-beta  | v0.2.0-alpha  | n/a          | n/a            |
-| **v0.1.0-alpha**     | v0.10.3-beta | v0.6.5-beta  | v0.2.0-alpha  | n/a          | n/a            |
+| LiT                   | LND          | Loop         | Faraday       | Pool         | Taproot Assets |
+|-----------------------|--------------|--------------|---------------|--------------|----------------|
+| **v0.15.0-alpha.rc2** | v0.19.1-beta | v0.31.1-beta | v0.2.16-alpha | v0.6.6-beta  | v0.6.0-rc3     |
```

Comment:
nit: Potentiaaaally we'd want to set `lnd` as `v0.19.1-rc1` here, but I'll leave that up to you to decide if we'd want to do that or not :)